pkgs/development/tools/analysis/hotspot/default.nix

```diff
@@ -64,6 +70,7 @@ mkDerivation rec {
 
   qtWrapperArgs = [
     "--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ rustc-demangle ]}"
```

Comment:
Do we still need LD_LIBRARY_PATH here?

Reply:
It does dlopen rustc_demangle. We could probably replace it with a patchelf call.